soh/src/code/z_play.c

```diff
@@ -12,11 +12,13 @@
 #include <overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h>
 #include "soh/Enhancements/enhancementTypes.h"
 #include "soh/Enhancements/game-interactor/GameInteractor_Hooks.h"
+#include "soh/Enhancements/stairs.h"
 
 #include <libultraship/libultraship.h>
 
 #include <time.h>
 #include <assert.h>
+// #include <stdlib.h>
```

Comment:
If this is truly not needed, should probably delete it instead of commenting it out.